interface-definitions/vpn-openconnect.xml.in

```diff
@@ -278,6 +293,26 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <leafNode name="tunnel-all-dns">
+                <properties>
+                  <help>If the tunnel-all-dns option is set to yes, tunnel all DNS queries via the VPN. This is the default when a default route is set.</help>
+                  <completionHelp>
+                    <list>yes no</list>
```

Comment:
I mentioned about `yes|no` before, it is bad CLI syntax, so we should avoid it (`yes|no, True|False, enable|disable, 1|0`) where it is possible
@dmbaturin What do you think?

Reply:
Please use a `<valueLess/>` node. We stopped using booleans. The new style is: If not is present -> true, otherwise false

Reply:
Roger